decentraland/explorer
# fix: material and shader caching issues

To test this branch:

- Check that setting different colors in the avatar editor changes color of the proper body part
- Go to ZONE:64,-64 turn back and check that the semi-transparent dome renders correctly
